cumulusci/tasks/util.py

```diff
@@ -149,14 +150,21 @@ def _run_task(self):
         kwargs = {}
         if "max" in self.options:
             kwargs["max"] = self.options["max"]
-        findReplace(
-            find=self.options["find"],
-            replace=self.options["replace"],
-            directory=self.options["path"],
-            filePattern=self.options["file_pattern"],
-            logger=self.logger,
-            **kwargs
-        )
+
+        # Split and trim file_pattern string into a list if not already a list
+        filePatterns = process_list_arg(self.options["file_pattern"])
+        if not len(filePatterns):
+            raise TaskOptionsError("At least one file_pattern must be specified.")
+
+        for fp in filePatterns:
+            findReplace(
+                find=self.options["find"],
+                replace=self.options["replace"],
+                directory=self.options["path"],
+                filePattern=fp,
+                logger=self.logger,
+                **kwargs
+            )
```

Comment:
It would be ideal to make findReplace handle a list of patterns itself so that we don't have to walk through the directories and write files multiple times. But I'm not worried about doing it this way.